Issue #41 link rel for coswid (#45)
ietf-rats/draft-birkholz-rats-corim: draft-birkholz-rats-corim.md

Privacy Considerations

> # Concise SWID Data Definition

Reviewer: ```suggestion
# Concise SWID Data Extensions
```

> # 2.7 The link-entry Map

Reviewer: Section numbering is created automatically.  At the MD level we only specify the heading nesting level by increasing the number of `#` signs prepended:

```suggestion
## The link-entry Map
```

> The link-entry map has been extended to define new relationship between a CoSWID and
> CoMID(identifies a hardware/firmware module).

Reviewer: ```suggestion
The link-entry map is extended to specify relationships between CoSWID and CoMID.
```

> When defining CoSWID to CoMID link entry map the following optional elements
> should not be used:
>  artifact
>  ownership
>  use

Reviewer: ```suggestion
When using CoSWID to CoMID relations, the following optional elements of the link-entry map MUST NOT used:

* artifact
* ownership
* use
```

# CoSWID Indexed Label Values

> # 4.4. Link Rel Values

Reviewer: Is this further section really needed?  I think this should be in the context of the section above (i.e., "Link-entry Map")

Author: This is how it is documented in CoSWID specification in an exact manner! Hence I replicated the tabular syntax from
Section 4.4 of CoSWID specification!

New indexes in the range 12 to 15 will be registered in the
IANA "Software Tag Link Relationship Values" registry defined in Section 6.2.7.

 | Index | Relationship Type Name | Definition                                     |
 --------|----------------------- |--------------------------------------------|
 | 12    | m-requires             | The link references a prerequisite module  |
 |       |                        | tag(s) for installing this software.       |
 | 13    | m-runs-on              | The link references a module tag that      |
 |       |                        | this software run on                       |
 | 14    | m-part-of              | The link references a module tag that this |
 |       |                        | software is part of                        |
 | 15    | m-supplemental         | The link references a module tag that the  |
 |       |                        | referencing tag supplements.               |

> 

Reviewer: This table should go in the IANA section below.  This section should, for each new relation defined, provide the semantic details.

Author: This is how it is documented in CoSWID specification in an exact manner! Hence I replicated the tabular syntax from
Section 4.4 of CoSWID specification!

> # 4.6 Link Privilege Values

Reviewer: Ditto

Author: Not applicable now!

New indexes for Software Tag Link Relationship Values needs to be registered as below:

 +=======+========================+============================================+
 | Index | Relationship Type Name | Description                                |
 +=======+========================+============================================+
 | 12    | needs                 | links to a module this software needs      |
 +-------+-----------------------+--------------------------------------------+
 | 13    | runs-on               | links to a module this software runs       |
 +-------+-----------------------+--------------------------------------------+
 | 14    | part-of               | links to a module this software is part of |
 +-------+-----------------------+--------------------------------------------+
 | 16    | m-supplemental        | provides additional information if a module|

>   +------+------------------------+-------------------------------------------+

Reviewer: This should be a [markdown table](https://www.markdownguide.org/extended-syntax/#tables), so something like:

```
| Index | Relationship Type | Definition |
|---
| 13 | runs-on | links to a CoMID tag describing the module on which this software runs |
...
{: #tbl-new-coswid-to-comid-link-ownership-values title="Link Ownership Values"}
```

See Body {{mybody}}.

> New indexes for Software Tag Link Relationship Values needs to be registered as below:

Reviewer: You also need to add the `trusted`/`untrusted` registration.

Author: Not Applicable now!
